brasilio_auth/templates/brasilio_auth/emails/password_reset_email.html

```diff
@@ -0,0 +1,11 @@
+Olá,
+
+Você está recebendo este email porque solocitou uma alteração de senha para o seu usuário no Brasil.io.
```

Comment:
Could you please replace with `Brasil.IO` (capitalized IO)? I think it creates a better identity of the name.

Reply:
:+1: